packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -218,3 +219,28 @@ async function oldPostsLastCommentedAt (post: DbPost) {
 }
 
 getCollectionHooks("Posts").editAsync.add(oldPostsLastCommentedAt)
+
+
+getCollectionHooks("Posts").newSync.add(async function FixEventStartAndEndTimes(post: DbPost): Promise<DbPost> {
```

Comment:
Are these really problems you've seen? Just curious.

Reply:
Nope, Jim and I were just talking about sorting events, and rather than account for the case where an event has no start date but has an end date we added this because we thought it was better